Signal/src/util/OWSBackupExport.h

```diff
@@ -0,0 +1,42 @@
+//
+//  Copyright (c) 2018 Open Whisper Systems. All rights reserved.
+//
+
+NS_ASSUME_NONNULL_BEGIN
+
+@class OWSBackupExport;
```

Comment:
`OWSBackupExport`  could be renamed `OWSBackupExportJob` if you like.  It represents a single attempt to backup the app's content.